src/api/app/assets/javascripts/webui/responsive_ux.js

```diff
@@ -22,3 +22,15 @@ $(function () {
     $('#username').focus();
   });
 });
+
+/* bootstrap's tabs javascript doesn't remove the active class
+   when tabs are implemented without the usage of ul and li */
```

Comment:
Have you considered to be a good open source citizen and open an issue over at bootstrap? 😄